web/src/app/projects/repository/repository.component.ts

```diff
@@ -9,6 +9,7 @@ import { RepositoryModel } from '../../shared/models/index.model';
   selector: 'dashboard-repository',
   templateUrl: './repository.component.html',
   styleUrls: ['./repository.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Is it possible to not use `none` and keep it default settings?

Reply:
Removed the none property.

Reply:
ok great 👍